.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+
+# Thanks for contributing a pull request! 
+## Please make sure you see our contribution guidelines: https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md
```

Comment:
We could make this visible only to the user?!

```suggestion
<!---
# Thanks for contributing a pull request! 
## Please make sure you see our contribution guidelines: https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md
-->
```

Reply:
Done.